docs/vsphere-golden-image.md

```diff
@@ -147,6 +147,9 @@ makes it ready for future deployments, maintaining all the changes needed for th
 You can find an example [unattend.xml](unattend.xml) file in this repository. It is just an example and, it **should 
 not** be used directly. You must customize the example to fit your needs.
 
+For example, the value of the `ComputerName` property in the [unattend.xml](unattend.xml) file must follows the
```

Comment:
Rather than `For example,`, you should say `Please note that the`. This is not an optional change but a must have. You should also call out that if the customer is apply Guest OS customization when launching from the resulting template, the name there again needs to match the k8s specs.